crossbundle/cli/src/commands/build/android.rs

```diff
@@ -329,11 +331,13 @@ impl AndroidBuildCommand {
         let mut aapt2_link =
             sdk.aapt2()?
                 .link_compiled_res(compiled_res, &apk_path, &manifest_path);
-        aapt2_link
-            .android_jar(sdk.android_jar(target_sdk_version)?)
-            .assets(context.android_assets().unwrap())
-            .proto_format(true)
-            .auto_add_overlay(true);
+        aapt2_link.android_jar(sdk.android_jar(target_sdk_version)?);
+        if let Some(assets) = context.android_assets() {
```

Comment:
This does return nothing from the `if` scope and why do we need `else`? Perhaps replace with:
```rust
if ... {
    aapt2_link.assets(assets)
} else {
    aapt2_link
}
```